crates/rune-macros/src/context.rs

```diff
@@ -493,6 +497,55 @@ impl Context {
 
                         attrs.install_with = Some(install_with);
                     }
+                    Meta(NameValue(syn::MetaNameValue {
+                        path,
+                        lit: Lit::Str(s),
+                        ..
+                    })) if path == DYNAMIC_FIELDS => {
+                        attrs.meta_fields = Some(match s.value().as_str() {
+                            "never" => format_ident!("Never"),
+                            "only" => {
+                                if cfg!(feature = "dynamic_fields") {
+                                    format_ident!("Only")
+                                } else {
+                                    self.errors.push(syn::Error::new_spanned(
+                                s,
+                                "attempted to set dynamic_fields to `only` while the feature `dynamic_fields` is disabled."
+                            ));
```

Comment:
Indentation here seems a bit off?

Reply:
Funnily enough it looked perfect until cargo fmt decided to make it look like that lol, I'll appease it shortly.